### PR TITLE
Job url

### PR DESCRIPTION
**Pull request by ludeknovy**
### 🤔 What's changed?

Added job url into execution summary

### ⚡️ What's your motivation? 

https://github.com/cucumber/react-components/issues/50

### 🏷️ What kind of change is this?

- :zap: New feature (non-breaking change which adds new behaviour)

### 📋 Checklist:

- [x] I agree to respect and uphold the [Cucumber Community Code of Conduct](https://cucumber.io/conduct/)
- [x] I've changed the behaviour of the code
  - [x] I have added/updated tests to cover my changes.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.
- [ ] Users should know about my change
  - [ ] I have added an entry to the "Unreleased" section of the [**CHANGELOG**](../CHANGELOG.md), linking to this pull request.

----

*This text was originally generated from a [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates), then edited by hand. [You can modify the template here.](https://github.com/cucumber/.github/edit/main/.github/PULL_REQUEST_TEMPLATE.md)*

### CODE REVIEW

**Review by aurelien-reeves — COMMENTED** [review]
Great, thanks! Looks pretty good!

Would it be possible to add labels right before the links to distinguish more easily the link to the commit from the link to the job?

File: src/components/app/ExecutionSummary.spec.tsx [hunk 1]
```diff
@@ -128,5 +128,11 @@ describe('ExecutionSummary', () => {
 
       expect(getByText('cucumber-js 8.0.0-rc.1')).toBeVisible()
     })
+    it('should include the job link', () => {
+      const { getByText } = render(<ExecutionSummary {...DEFAULT_PROPS} />)
+      const jobLinkElement = getByText(DEFAULT_PROPS.meta?.ci?.buildNumber as string)
+      assert.ok(jobLinkElement)
```
Comment (aurelien-reeves) [inline src/components/app/ExecutionSummary.spec.tsx:134]: We are actually using the jest expectations

**Comment by ludeknovy** [comment]
@aurelien-reeves Sure, I was experimenting with different icons, such as arrows spin for the job url, but it didnt feel like 100%. What labels - `Git ref` and `Job`?

File: src/components/app/ExecutionSummary.spec.tsx (hunk 1 above)
Comment (ludeknovy) [reply src/components/app/ExecutionSummary.spec.tsx:134]: Yeah, sorry, started working on it on stale `main` 🤦

Comment (ludeknovy) [reply src/components/app/ExecutionSummary.spec.tsx]: Updated.

**Comment by aurelien-reeves** [comment]
> `Git ref` and `Job`?

Yeap, looks good :)

**Comment by ludeknovy** [comment]
Current look with labels:

Screenshot 2022-04-19 at 11 17 16

**Review by aurelien-reeves — APPROVED** [review]
Looks good to me, thanks @ludeknovy 

@davidjgoss any comment?

**Review by davidjgoss — APPROVED** [review]

**Pull request merged**